pubspec.yaml

```diff
@@ -26,6 +26,7 @@ dependencies:
   image_picker: ^0.6.7+21
   path_provider: ^1.3.0
   shared_preferences: 0.5.12+4
+  logging: ^0.11.4
```

Comment:
Update Libraries.txt

Reply:
Libraries.txt should only contain libs of the SDK, not of the app.

Reply:
made a copy of the library file from  flutter sdk